tests/app/test_app.py

```diff
@@ -392,6 +377,7 @@ def test_case_service_called_on_sent_if_NotifyCaseService_is_set(self, case):
 
         self.app.config["NOTIFY_VIA_GOV_NOTIFY"] = '0'
         self.app.config['NOTIFY_CASE_SERVICE'] = '1'
+        self.app.config['NOTIFY_VIA_GOV_NOTIFY'] = '0'
```

Comment:
This is the same as one of the lines above.

Reply:
Yep , came in during a merge